content/workflows/policies/csp/aws/service/iam/resources/policy/iam-role-not-enabled.yaml

```
@@ -0,0 +1,20 @@
+schema: 1.0
+labels:
+  - contextualization
+type: contextualize
+category: iam
+subcategory: authorization
+id: "iam-role-not-enabled"
```

```suggestion
id: "iam-role-not-enabled-for-ec2"
```

Please also rename the file